src/web/app/pages-help/instructor-help-page/instructor-help-courses-section/instructor-help-courses-section.component.ts

```diff
@@ -14,11 +15,54 @@ export class InstructorHelpCoursesSectionComponent extends InstructorHelpSection
 
   readonly supportEmail: string = environment.supportEmail;
 
-  constructor() {
+  isCollapsed: boolean = false;
+  isCollapsed1: boolean = false;
+  isCollapsed2: boolean = false;
+  isCollapsed3: boolean = false;
+  isCollapsed4: boolean = false;
+  isCollapsed5: boolean = false;
+  isCollapsed6: boolean = false;
+  isCollapsed7: boolean = false;
+  isCollapsed8: boolean = false;
+  isCollapsed9: boolean = false;
+  isCollapsed10: boolean = false;
+  isCollapsed11: boolean = false;
+  isCollapsed12: boolean = false;
+  isCollapsed13: boolean = false;
+  isCollapsed14: boolean = false;
+  isCollapsed15: boolean = false;
+  isCollapsed16: boolean = false;
+  isCollapsed17: boolean = false;
+  isCollapsed18: boolean = false;
+  isCollapsed19: boolean = false;
```

Comment:
Can we improve the naming?

Reply:
Same for multiple places

Reply:
Sorry, sure. I was hesitant to modify the variable names that had been there before. 

Changes now made, please see the last commits below. I will be happy to work further on naming if needed.